include/kunit

```diff
@@ -180,6 +180,8 @@ need_kconfig:
 
 # below are kunit test configs of bool type
 - TSNEP_SELFTESTS: y
+- HAVE_STATIC_CALL: y
+- STATIC_CALL_SELFTEST: y
```

Comment:
help add empty lines before/after these 2 lines to view them as a group (kconfig and its dependency)

Reply:
ok!